doc/man7/OSSL_STORE-winstore.pod

```diff
@@ -0,0 +1,86 @@
+=pod
+
+=head1 NAME
+
+OSSL_STORE-winstore - OpenSSL built in OSSL_STORE for Windows
```

Comment:
Isn't this only "built-in" to the default and base providers?

We should probably link this page from OSSL_PROIVDER-default.pod and OSSL_PROVIDER-base.pod

Reply:
Ah, yes, I forgot to link.